docs/guides/dns/unbound.md

```diff
@@ -183,13 +183,13 @@ Finally, configure Pi-hole to use your recursive DNS server by specifying `127.0
 
 (don't forget to hit Return or click on `Save`)
 
-### Disable `resolvconf` for `unbound` (optional)
+### Disable `resolvconf` for `unbound` (optional for some, required for Bullsye+ releases)
 
-The `unbound` package can come with a systemd service called `unbound-resolvconf.service` and default enabled.
-It instructs `resolvconf` to write `unbound`'s own DNS service at `nameserver 127.0.0.1` , but without the 5335 port, into the file `/etc/resolv.conf`.
-That `/etc/resolv.conf` file is used by local services/processes to determine DNS servers configured.
-If you configured `/etc/dhcpcd.conf` with a `static domain_name_servers=` line, these DNS server(s) will be ignored/overruled by this service.
+Recent OS releases of the `unbound` package depends on a package called openresolv, which has a configuration file (/etc/resolvconf.conf) and a systemd service called `unbound-resolvconf.service.` You need to edit the configuration file and disable the service to work-around the bug below.
```

Comment:
The `unbound` package does not depend on `openresolv`. Furthermore all dependants depend on any "resolvconf" implementation, which includes the `resolvconf` package. So either replacing `openresolv` with `resolvconf` or simply purging it is another simple solution.

Reply:
Thank you @MichaIng. Given the wide userbase, I think removing openresolv might end up with more questions from other things not working.  I've addressed this in the updated PR.

Reply:
There is really nothing `openresolv` does to a DNS server system than breaking it. This package is explicitly aiming to configure all known resolvers, including Unbound, to use the host systems DNS, which in every case either breaks it (if the system uses Unbound itself on another port than 53), creates a DNS loop (if the system itself uses Unbound on port 53) or breaks the purpose of Unbound (being a recursive resolver or using DoT or DoH) if the system uses an upstream provider. It is meant for cases where Unbound is used by the host system only as DNS cache or similar, which is nearly never the case, and definitely not for Pi-hole users where Unbound is implicitly for the LAN, not for the host system (only). So while replacing it is the more intrusive method, it is also simpler and cleaner.

The whole problem is btw caused by Debian (implying Ubuntu), while the upstream Unbound default config has all additional resolver setups disabled by default. They only want to give the option, in case anyone really uses it as sort of DNS cache or similar for the host system only. Debian for whatever reason enables all those additional resolver setups by default.

But whether or not to mention additional solution (yours is working as well, of course), more important is the false first sentence. Neither Unbound nor any Pi-hole dependency depends on `openresolv` or `resolvconf`. Those are anyway mainly used (optionally) in combination with a DHCP client, while with Pi-hole you want a static IP. `openresolv` is simply pre-installed on most Debian and Ubuntu images, that's all.

Reply:
Ah I see, you adjusted it already. All correct now and good to have option 1 mentioned 👍.